src/main/java/datawave/data/normalizer/NumberNormalizer.java

```diff
@@ -10,6 +10,14 @@ public class NumberNormalizer extends AbstractNormalizer<BigDecimal> {
     private static final long serialVersionUID = -2781476072987375820L;
     
     public String normalize(String fv) {
+        if (NumericalEncoder.isPossiblyEncoded(fv)) {
+            try {
+                NumericalEncoder.decode(fv);
+                return fv;
+            } catch (Exception e2) {
+                // no problem here, we will simply try to encode it below
+            }
+        }
         try {
```

Comment:
Perfect, this addresses the issue of threads consistently filling in stacktraces.

It looks like we could also take advantage [of this constructor](https://docs.oracle.com/javase/8/docs/api/java/lang/Throwable.html#Throwable-java.lang.String-java.lang.Throwable-boolean-boolean-) to avoid filling in stacktraces altogether.